### PR TITLE
Map 28 support portal calls subsites using mi

### PR DESCRIPTION
**Pull request by kola-samuel**

### CODE REVIEW

File: src/SFA.DAS.Support.Indexer.ApplicationServices.UnitTests/IndexResourceProcessorBase.cs [hunk 1]
```diff
@@ -31,15 +30,17 @@ public class IndexResourceProcessorBase
...
         protected const int _indexToRetain = 2;
         protected IEnumerable<AccountSearchModel> _accountModels;
         protected SiteResource _accountSiteResource;
         protected SiteResource _userSiteResource;
 
-
         protected void Initialise()
         {
             _baseUrl = new Uri("http://localhost");
+            _resourceIdentifier = "https://citizenazuresfabisgov.onmicrosoft.com/das-at-test-as-ar";
```
Comment (cofaulco) [inline src/SFA.DAS.Support.Indexer.ApplicationServices.UnitTests/IndexResourceProcessorBase.cs:43]: don't think this real identifier should be hardcoded into the source code

File: src/SFA.DAS.Support.Indexer.Worker/WebConfiguration.cs [hunk 2]
```diff
@@ -2,15 +2,17 @@
...
 using SFA.DAS.Support.Indexer.ApplicationServices.Settings;
 using SFA.DAS.Support.Shared.SiteConnection;
+using System.Collections.Generic;
 
 namespace SFA.DAS.Support.Indexer.Worker
 {
     public class WebConfiguration : IWebConfiguration
     {
-        [JsonRequired] public SiteConnectorSettings SiteConnector { get; set; }
+        public SiteConnectorSettings SiteConnector { get; set; }
```
Comment (cofaulco) [inline src/SFA.DAS.Support.Indexer.Worker/WebConfiguration.cs:11]: can this be removed now?

File: src/SFA.DAS.Support.Indexer.Worker/DependencyResolution/DefaultRegistry.cs [hunk 3]
```diff
@@ -29,19 +29,21 @@ public DefaultRegistry()
                 x.GetInstance<IRequestContext>(),
                 x.GetInstance<ILoggingPropertyFactory>().GetProperties())).AlwaysUnique();
 
-
             var configuration = GetConfiguration();
             For<IWebConfiguration>().Use(configuration);
             For<ISearchSettings>().Use(configuration.ElasticSearch);
             For<ISiteSettings>().Use(configuration.Site);
-            For<ISiteConnectorSettings>().Use(configuration.SiteConnector);
+            For<ISiteConnectorSettings>().Use(configuration.SiteConnector ?? new SiteConnectorSettings());
```
Comment (cofaulco) [inline src/SFA.DAS.Support.Indexer.Worker/DependencyResolution/DefaultRegistry.cs:36]: can this be removed now?

Comment (kola-samuel) [reply src/SFA.DAS.Support.Indexer.Worker/DependencyResolution/DefaultRegistry.cs:36]: leaving it there incase this get released without MI been setup yet.

File: src/SFA.DAS.Support.Indexer.Worker/WebConfiguration.cs (hunk 2 above)
Comment (kola-samuel) [reply src/SFA.DAS.Support.Indexer.Worker/WebConfiguration.cs:11]: leaving it there incase this get released without MI been setup yet once released can remove it

File: src/SFA.DAS.Support.Indexer.ApplicationServices.UnitTests/IndexResourceProcessorBase.cs (hunk 1 above)
Comment (kola-samuel) [reply src/SFA.DAS.Support.Indexer.ApplicationServices.UnitTests/IndexResourceProcessorBase.cs]: changed

File: src/SFA.DAS.Support.Indexer.Worker/DependencyResolution/DefaultRegistry.cs (hunk 3 above)
Comment (cofaulco) [reply src/SFA.DAS.Support.Indexer.Worker/DependencyResolution/DefaultRegistry.cs:36]: this branch and its config branch will only ever get merged at the same time so this isn't an option.

File: src/SFA.DAS.Support.Indexer.Worker/WebConfiguration.cs (hunk 2 above)
Comment (cofaulco) [reply src/SFA.DAS.Support.Indexer.Worker/WebConfiguration.cs:11]: this branch and its config branch will only ever get merged at the same time so this isn't an option.

File: src/SFA.DAS.Support.Indexer.Worker/DependencyResolution/DefaultRegistry.cs (hunk 3 above)
Comment (kola-samuel) [reply src/SFA.DAS.Support.Indexer.Worker/DependencyResolution/DefaultRegistry.cs]: removed

File: src/SFA.DAS.Support.Indexer.Worker/WebConfiguration.cs (hunk 2 above)
Comment (kola-samuel) [reply src/SFA.DAS.Support.Indexer.Worker/WebConfiguration.cs]: removed

**Review by cofaulco — APPROVED** [review]

**Pull request merged**